src/Scientist/Internals/Experiment.cs

```diff
@@ -68,20 +69,19 @@ public ExperimentInstance(string name, Func<Task<T>> control, Func<Task<T>> cand
 
         static async Task<ExperimentResult> Run(Func<Task<T>> experimentCase)
         {
-            var sw = new Stopwatch();
-            sw.Start();
+            Chrono chrono = new Chrono();
```

Comment:
Why would we  instantiate `chrono` here and then pass it into a method that takes it as an `out` parameter?